Frontend/android-native/app/src/user/jatriSaathi/assets/juspay/v1-configuration.js

```diff
@@ -242,6 +242,7 @@ window.getMerchantConfig = function () {
   , "callOptions" : ["ANONYMOUS"]
   , "showNearByDrivers" : true
   , "showDisabilityBanner" : false
+  , "enableWhatsappOTP" : ["BD"]
```

Comment:
can we remove this from `v1-configuration` and only add in Default config with Country code